src/providers/openrouter/chatComplete.ts

```diff
@@ -81,6 +81,7 @@ export const OpenrouterChatCompleteResponseTransform: (
   response: OpenrouterChatCompleteResponse | OpenrouterErrorResponse,
   responseStatus: number
 ) => ChatCompletionResponse | ErrorResponse = (response, responseStatus) => {
+  console.log('response:', response);
```

Comment:
Sorry for the nitpick: Can you please remove console logs from the PR?

Reply:
sorry for my negligence